subsys/usb/device/usb_device.c

```diff
@@ -508,6 +513,38 @@ static bool usb_get_descriptor(struct usb_setup_packet *setup,
 	return found;
 }
 
+/*
+ * @brief Get 32-bit endpoint mask from index
+ *
+ * In the returned 32-bit word, the bit positions in the lower 16 bits
+ * indicate OUT endpoints, while the upper 16 bits indicate IN
+ * endpoints
+ *
+ * @param [in]  ep Endpoint of interest
+ *
+ * @return 32-bit bitmask
+ */
+static uint32_t get_ep_mask_from_addr(uint8_t ep)
+{
+	uint32_t ep_mask = 0;
```

Comment:
I suggest to use ep_bm consistently.

Reply:
Changed (I also changed the name of the function)